src/pudl/analysis/state_demand.py

```diff
@@ -472,7 +473,10 @@ def _out_ferc714__hourly_demand_matrix(
     return df
 
 
-@asset(compute_kind="Python")
+@asset(
+    compute_kind="Python",
+    op_tags={"memory-use": "high"},
+)
 def _out_ferc714__hourly_imputed_demand(
```

Comment:
I added a tag to this asset because it is parallelized internally and gobbles up a lot of memory and cpus.

Reply:
I'll try running the full ETL locally and see what this does to the overall run time.  I'm worried that because this process is running for almost the entire duration of the ETL, having it tagged as high memory usage will mean that the EPA CEMS ends up taking 2x as long as it does now.

When I ran this process in isolation locally it used 4-6 CPUs, but only 1-2 GB of memory, so I think we might not need for it to be held out as a high memory use asset.

Reply:
Ah ok. It also wasn't running when the VM failed so I'm down to not restrict it's concurrency.

Reply:
I'm hunting down a bunch of other high memory assets now. There are some doozies! The EIA930 tops out at 11GB, and the EIA860M changelog table at 8GB.